core/src/main/java/com/scalar/db/storage/dynamo/DynamoTableMetadataManager.java

```diff
@@ -25,11 +47,14 @@
  */
```

Comment:
It's basically rewritten, so please put your name on it!

Reply:
Updated in [90037fd]( https://github.com/scalar-labs/scalardb/pull/301/commits/90037fdc8400d01e7387d17c33f4069dc9c02c9a)